### DISCUSSION TITLE
what is the "Windows Power mode" and should i mess arround with it?

### DISCUSSION THREAD

**Discussion by Nuke10** [seerge/g-helper#2108] [Q&A]
so i decided to take a look into G-helper, so far i like how its works.
but there are few things i found confusing,

in this case: 
when i go to: Fans+Power i noticed there is a drop down menu that said: "Windows Power mode" that have a few options.

so my questions are. what is it. and, can i play around with it without accidently endangering my laptop's warranty?

**Comment by Kikimahe** [comment]
Hi, 

Windows power modes in Ghelper correspond to Windows power modes in Windows. (screenshoot) :)
Ghelper switches between these modes when you select Silent, Balanced or Turbo.

Off course, Ghelper is safe. It uses the same settings as Armory Crate.
You can modify whatever you want. ;-)

![Capture d'écran 2024-02-24 082748](https://github.com/seerge/g-helper/assets/142881210/1bd2bfb0-e0a3-4e8b-8f55-766bb4d6c6b8)

**Reply by Nuke10** [reply]
Just wanted to make sure.thanks
